v3/generator/operations/operations.go

```diff
@@ -30,22 +30,36 @@ func Generate(doc libopenapi.Document, path, packageName string) error {
 	
 import (
 	"bytes"
+	"context"
+	"crypto/hmac"
+	"crypto/sha256"
+	"encoding/base64"
 	"encoding/json"
+	"errors"
 	"fmt"
 	"io"
 	"net"
 	"net/http"
-	"context"
+	"net/http/httputil"
 	"net/url"
+	"os"
+	"sort"
+	"strings"
 	"time"
 
-	"github.com/google/uuid"
 	"github.com/go-playground/validator/v10"
+	"github.com/google/uuid"
 )
 `, packageName))
 
+	h, err := os.ReadFile("./operations/helpers.tmpl")
+	if err != nil {
+		return err
+	}
+	output.Write(h)
```

Comment:
I find it slightly weird to have a `.tmpl` that is actually not templated but pure go code.

Reply:
https://github.com/exoscale/egoscale/pull/611/commits/81a4a65fdc9965f271ee57d10db2276929a28c6e